.github/workflows/build.yml

```diff
@@ -35,6 +35,9 @@ jobs:
       - name: Build source
         run: yarn build
 
+      - name: Build flow types
+        run: yarn build:types-flow
```

Comment:
Normally the generated types are still committed to the repository, so that you can visually inspect the generated output or manually modify them. I think that would be good to have, in addition to having a build step validate that the current types are up to date with the generated types. This might complicate the process too much, but figured I would mention it anyway.

Reply:
sounds good to me. I'll see if I can chuck it in the pre-commit hook as well.

Reply:
Didn't end up putting it in the pre-commit hook yet so the scope doesn't blow out too much (type generation is slow so I'd need to make sure the staged changes are only applied) but will consider doing this later in a future PR if the dev loop suffers too much